src/components/ScrollTrack/equalWidthTrack.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
```

Comment:
Can we add a comment block to the top of this file to describe it a bit more? When/why to use it, for example. We should add similar documentation in the ScrollTrack markdown file.